src/snowflake/snowpark/functions.py

```diff
@@ -3836,14 +4083,27 @@ def array_contains(variant: ColumnOrName, array: ColumnOrName) -> Column:
 
     Args:
         variant: Column containing the VARIANT to find.
-        array: Column containing the ARRAY to search."""
+        array: Column containing the ARRAY to search.
+
+    Example::
+        >>> from snowflake.snowpark import Row
+        >>> df = session.create_dataframe([Row([1, 2]), Row([1, 3])], schema=["a"])
+        >>> df.select(array_contains(lit(2), "a").alias("result")).show()
+        ------------
+        |"RESULT"  |
+        ------------
+        |True      |
+        |False     |
+        ------------
+        <BLANKLINE>
+    """
     v = _to_col_if_str(variant, "array_contains")
     a = _to_col_if_str(array, "array_contains")
     return builtin("array_contains")(v, a)
 
 
 def array_insert(
-    array: ColumnOrName, pos: ColumnOrName, element: ColumnOrName
+    array: ColumnOrName, pos: Union[ColumnOrName], element: ColumnOrName
```

Comment:
Shouldn't `Union` have two different types here. `ColumnOrName` is already a union of `Column` and `str` I believe

Reply:
Good catch. I was thinking something then left it there.